src/server.js

```diff
@@ -78,7 +78,7 @@ function get_token_info (access_token) {
 
 function check_token_validity ({result, access_token, required_scopes}) {
     return new Promise((resolve, reject) => {
-        if (!result.scopes) {
+        if (!_.has(result, 'scopes')) {
```

Comment:
But doesn't this throw error when there are no scopes at all in the result (which is what PM would be using?)

Reply:
There's always a scopes field regardless if the access token is generated with or without scopes.

If PM requested without a scope, the error will say that it needs a required scope (specifying the scope needed by the route).